tools.yml

```diff
@@ -10,6 +12,8 @@ tools:
     rules: []
     rank: |
       helpers.weighted_random_sampling(candidate_destinations)
+  default:
+    extends: tpvdb_default
```

Comment:
if default inherits tbvdb_default, what do this do that is different?

Reply:
The idea was to preserve backward compatibility, but I think that this may end up just creating an infinite loop. We probably want this these changes to spread to all destinations anyway, so I'll change this back.

Reply:
Fixed.